### ISSUE TITLE
[jss-plugin-expand] Border not expanding correctly

### ISSUE THREAD

**Issue by YassienW** [cssinjs/jss#1323]
__Expected behavior:__

As per https://cssinjs.org/jss-plugin-expand?v=v10.0.0-alpha.9

```
container: {
    border: {
      color: 'black',
      width: 1,
      style: 'solid'
    }
  }
```
should compile to
```
.container {
  border: 1px solid black;
}
```

__Describe the bug:__

It compiles to
```
.container {
  border-width: 1px;
  border-style: solid;
  border-color: black;
}
```

__Codesandbox link:__ 
https://codesandbox.io/s/jss-plugin-expand-issue-ww682

__Versions (please complete the following information):__
 - jss: 10.1.1
 - Browser [e.g. chrome, safari]: Chrome
 - OS [e.g. Windows, macOS]: Linux
Feel free to add any additional versions which you may think are relevant to the bug.

This was briefly mentioned in #1023

**Comment by kof** [comment]
Why do you consider this to be a bug? This is valid CSS

```csss
.container {
  border-width: 1px;
  border-style: solid;
  border-color: black;
}
```

**Comment by YassienW** [comment]
I thought a one liner would be expected and is a feature of this plugin as per the documentation:
https://cssinjs.org/jss-plugin-expand?v=v10.0.0-alpha.9

If not i guess the documentation should be updated for consistency, i wasn't entirely sure if i was doing something wrong or if the documentation was incorrect

**Comment by kof** [comment]
yeah, docs is wrong there, though I don't remember exactly why we don't generate a shortcut in this case

**Comment by typical000** [comment]
As far as I remember the reason was that it's more "safe" and optimal in terms of performance to write all rules separately instead of one shortcut concatenating sting in one line.
As "safe" I mean that there is no standard in CSS in which order we should write rules inside shorthand. For example if we use padding and concatenate them without any additional checks:
```
// JS
container: {
  padding: {
    top: 10,
    left: 1,
    right: 20
  }
}

// Resulting CSS
.container {
  padding: 10px 1px 20px; /* Wrong! */
}
```
For avoiding such bug we should write additional logic for making shorthand rule if all values are correct, or write all rules separately it there is something wrong.

**Comment by kof** [comment]
That makes sense @typical000, thanks!
